Intersections_3/test/Intersections_3/do_intersect_3.cpp

```diff
@@ -0,0 +1,114 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
+typedef K::Triangle_3 Triangle_3;
+typedef K::Segment_3 Segment_3;
+typedef K::Ray_3 Ray_3;
+typedef K::Line_3 Line_3;
+typedef K::Tetrahedron_3 Tetrahedron_3;
+typedef K::Point_3 Point_3;
+typedef K::Plane_3 Plane_3;
+typedef K::Iso_cuboid_3 Iso_cuboid_3;
+typedef K::Sphere_3 Sphere_3;
+
+int main()
+{
+  Point_3 p(0,0,0), q(1,1,1), r(1,0,0), s(1,1,0);
+  Segment_3 s0(p,q),s1(p,q);
+  Ray_3 ray(p,q);
+  Line_3 lin(p,q);
+  Triangle_3 t0(p,q,r), t1(p,q,r);
+  Tetrahedron_3 tet(p,q,r,s);
+  Plane_3 pla(p,q,r);
+  Iso_cuboid_3 ic(p,q);
+  Sphere_3 sp(p,1);
+  CGAL::Bbox_3 bb;
+
+  do_intersect(p,sp);
+  do_intersect(bb,t0);
```

Comment:
- `bb` is not initialized
- there is already `test_intersection_3.cpp` that checks `do_intersect` and API calls, why a new file with only a subset of the supported combinations?

Reply:
bb is initialized to a bbox with [ + inf , - inf ] ^3
Is it only a subset?  I had started this file to add tests little by little, just to check that it compiles when divsion enabled. Shall I only keep the triple plane test here?

Reply:
Just counting the lines, there are fewer tests than in https://github.com/CGAL/cgal/blob/master/Intersections_3/test/Intersections_3/test_intersections_3.cpp#L1232. 

Planex3 has its function: https://github.com/CGAL/cgal/blob/master/Intersections_3/test/Intersections_3/test_intersections_3.cpp#L223.

Reply:
I just removed the additional file.